antlr-kotlin-examples-jvm/.gitignore

```diff
@@ -1 +0,0 @@
-generated-src
```

Comment:
why?

Reply:
the gradle default is to put this into "antlr-kotlin-examples-jvm/build/generated-src/antlr/main", the build folder is already ignored by git